src/client/pydaos/raw/daos_api.py

```diff
@@ -1389,24 +1437,101 @@ def create(self, poh, con_uuid=None, cb_func=None):
             self.uuid = None
         else:
             conversion.c_uuid(con_uuid, self.uuid)
-
         self.poh = poh
+        if con_prop is not None:
+            self.cont_input_values = con_prop
+        # We will support only basic properties. Full
+        # container properties will not be exposed.
+        # Create DaosProperty for checksum
+        # 1. Layer Type.
+        # 2. Enable checksum,
+        # 3. Server Verfiy
+        # 4. Chunk Size Allocation.
+        if ((self.cont_input_values.type != "Unknown")
+                and (self.cont_input_values.enable_chksum is False)):
+            # Only layer_type like posix, hdf5 defined.
+            num_prop = 1
+        elif ((self.cont_input_values.type == "Unknown")
+                and (self.cont_input_values.enable_chksum is True)):
+            # Obly checksum enabled.
+            num_prop = 3
+        elif ((self.cont_input_values.type != "Unknown")
+                and (self.cont_input_values.enable_chksum is True)):
+            # Both layer and checksum properties define
+            num_prop = 4
+
+        if ((self.cont_input_values.type != "Unknown")
+                or (self.cont_input_values.enable_chksum is True)):
+            self.cont_prop = daos_cref.DaosProperty(num_prop)
+        # idx index is used to increment the dpp_entried array
+        # value. If layer_type is None and checksum is enabled
+        # the index will vary. [eg: layer is none, checksum
+        # dpp_entries will start with idx=0. If layer is not
+        # none, checksum dpp_entries will start at idx=1.]
+        idx = 0
+        if self.cont_input_values.type != "Unknown":
+            self.cont_prop.dpp_entries[idx].dpe_type = ctypes.c_uint32(
+                DaosContPropEnum.DAOS_PROP_CO_LAYOUT_TYPE.value)
+            if self.cont_input_values.type == "posix":
+                self.cont_prop.dpp_entries[idx].dpe_val = ctypes.c_uint64(
+                    DaosContPropEnum.DAOS_PROP_CO_LAYOUT_POSIX.value)
+            elif self.cont_input_values.type == "hdf5":
+                self.cont_prop.dpp_entries[idx].dpe_val = ctypes.c_uint64(
+                    DaosContPropEnum.DAOS_PROP_CO_LAYOUT_HDF5.value)
+            else:
+                self.cont_prop.dpp_entries[idx].dpe_val = ctypes.c_uint64(
+                    DaosContPropEnum.DAOS_PROP_CO_LAYOUT_UNKOWN.value)
```

Comment:
(pylint-no-member) Instance of 'Enum' has no 'DAOS_PROP_CO_LAYOUT_UNKOWN' member